src/odemis/util/registration.py

```diff
@@ -358,6 +363,13 @@ def _cluster_edges(
         matched centroids.
 
     """
+    n_edges = graph.size(False)
```

Comment:
Let's make it more explicit:
```suggestion
    n_edges = graph.size(directed=False)
```

Reply:
done